py3status/modules/xrandr.py

```diff
@@ -37,15 +37,16 @@
         (default '=')
     icon_extend: icon used to display a 'extend' combination
         (default '+')
+    on_udev_drm: dynamic variable to watch for `drm` udev subsystem events to
+        trigger specified action.
+        (default 'refresh_and_freeze')
```

Comment:
(Opinion) `on_udev` prefix sucks a little. I am thinking something like `udev_subsystem_drm  = 'freeze'` or `udev_event_drm = 'refresh_and_freeze'`.

I admit I don't like `refresh_and_freeze` param... It could be `refresh`, `always` or `refresh`, `async`, but I'm not sure if my ideas are any better.

Reply:
the reason for the naming is that it is similar to `on_click` but instead of a click action it is a `udev` action.

I don't like `refresh_and_freeze` either but it does allow us to get the new  xrandra module niceness to the masses.

Reply:
Same, I'd stick with this in line with on_click please

Reply:
It's not `on_udev` itself that sucks... It's suffixing `on_udev` with something not obvious to the users that sucks a litlte as it does not look that clean. I feel using a list of tuples similar to `thresholds` might fare a bit better... like this:
```python
xrandr {
    on_udev = [('drm', 'refresh_and_freeze')]
}
```
We could find a better alias for `refresh_and_freeze` in the future... Could be `async`, `track`, etc.